datajunction-server/alembic/versions/2025_01_22_1638-0e428c1bb9bc_notification_preferences_table.py

```diff
@@ -0,0 +1,84 @@
+"""notification_preferences table
+
+Revision ID: 0e428c1bb9bc
+Revises: 70904373eab3
+Create Date: 2025-01-22 16:38:57.407898+00:00
+
+"""
+# pylint: disable=no-member, invalid-name, missing-function-docstring, unused-import, no-name-in-module
+
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "0e428c1bb9bc"
+down_revision = "70904373eab3"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "notification_preferences",
```

Comment:
nice! we need more tables with underscores in names :)

Reply:
You're lucky I didn't name it `user_notification_preferences` 😂

Reply:
Actually I was serious. I think most of our tables are worddelimiterless.

Reply:
Haha, yeah, I think it started off that way due to the default table name generation pattern if you don't explicitly spell out the table name, and that has since calcified...